c2corg_ui/static/js/documentdetailsview.js

```diff
@@ -68,6 +90,47 @@ app.ViewDetailsController.prototype.openModal = function(selector) {
   this.modal_.open({animation: true, size: 'lg', template: template});
 };
 
+
+/**
+ * @param {Date} dateStart
+ * @param {Date} dateEnd
+ * @suppress {checkTypes} it told (found scope | null, required -> scope) at compile()(this.scope)...
```

Comment:
You can not never rely on `@suppress {checkTypes}` to silent the compiler.
The only appropriate way is to fix the underlying issue:
```diff
diff --git a/c2corg_ui/static/js/documentdetailsview.js b/c2corg_ui/static/js/documentdetailsview.js
index 35857ce..2d39c1e 100644
--- a/c2corg_ui/static/js/documentdetailsview.js
+++ b/c2corg_ui/static/js/documentdetailsview.js
@@ -47,7 +47,7 @@ app.module.directive('appViewDetails', app.viewDetailsDirective);
 
 /**
  * @param {Object} $uibModal modal from angular bootstrap
- * @param {angular.Scope} $scope Scope.
+ * @param {!angular.Scope} $scope Scope.
  * @param {angular.$compile} $compile Angular compile service.
  * @constructor
  * @export
@@ -67,7 +67,7 @@ app.ViewDetailsController = function($uibModal, $compile, $scope) {
   this.compile_ = $compile;
 
   /**
-   * @type {angular.Scope}
+   * @type {!angular.Scope}
    * @private
    */
   this.scope_ = $scope;
@@ -93,7 +93,6 @@ app.ViewDetailsController.prototype.openModal = function(selector) {
 /**
  * @param {Date} dateStart
  * @param {Date} dateEnd
- * @suppress {checkTypes} it told (found scope | null, required -> scope) at compile()(this.scope)...
  * @export
  */
 app.ViewDetailsController.prototype.displayDate = function(dateStart, dateEnd) {
@@ -125,7 +124,7 @@ app.ViewDetailsController.prototype.displayDate = function(dateStart, dateEnd) {
                 ' - <span class="date-locale">{{\' ' + dateEnd.toISOString() + ' \'  | amDateFormat:"Do MMMM YYYY" }})</span>')(this.scope_);
       }
     }
-    $('.outing-date').append(dateHTML);
+    $('.outing-date').append(dateHTML.get());
   }
 };
 
diff --git a/c2corg_ui/static/js/documentediting.js b/c2corg_ui/static/js/documentediting.js
index 1ca4076..cbc2f67 100644
--- a/c2corg_ui/static/js/documentediting.js
+++ b/c2corg_ui/static/js/documentediting.js
@@ -199,7 +199,6 @@ app.DocumentEditingController = function($scope, $element, $attrs,
 /**
  * @param {Object} response Response from the API server.
  * @private
- * @suppress {checkTypes}
  */
 app.DocumentEditingController.prototype.successRead_ = function(response) {
   var data = response['data'];
```